setup.py

```diff
@@ -48,115 +39,4 @@ def run_tests(self):
         sys.exit(errno)
 
 
-test_path = os.path.join(root_path, 'odl', 'test')
-
-
-def find_tests():
-    """Discover the test files for packaging."""
-    tests = []
-    for path, _, filenames in os.walk(os.path.join(root_path, test_path)):
-        for filename in filenames:
-            basename, suffix = os.path.splitext(filename)
-            if (
-                suffix == '.py'
-                and (basename.startswith('test_')
-                     or basename.endswith('_test'))
-            ):
-                tests.append(os.path.join(path, filename))
-
-    return tests
-
-
-# Determine version from top-level package __init__.py file
-with open(os.path.join(root_path, 'odl', '__init__.py')) as f:
-    for line in f:
-        if line.startswith('__version__'):
-            version = line.strip().split()[-1][1:-1]
-            break
-
-
-long_description = """
```

Comment:
This text will be lost. Feature or bug?

Reply:
Now the `README.md` file [will be picked](https://github.com/odlgroup/odl/blob/405a151a5daabdbb6b432e62cfeac189ec2930c3/setup.cfg#L5-L6) instead. That saves us from keeping both in sync.